crates/rolldown/tests/rolldown/errors/parse_error/invalid_define_value/artifacts.snap

```diff
@@ -0,0 +1,12 @@
+---
+source: crates/rolldown_testing/src/integration_test.rs
+snapshot_kind: text
+---
+# Errors
+
+## PARSE_ERROR
+
+```text
+[PARSE_ERROR] Error: Parse failed, got: "Invalid assignment in object literal"
```

Comment:
I guess we should point the error is related the `define` ponfig.

Reply:
Yes, it's not very intuitive at the moment. I will improve it.